frontend/src/components/QuestionsList.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { Progression, Question } from '../api/models'
+import { Divider } from '@equinor/eds-core-react'
+import { Box } from '@material-ui/core'
+import QuestionAndAnswerFormWithApi from './QuestionAndAnswer/QuestionAndAnswerFormWithApi'
+import QuestionActionsListWithApi from './Action/QuestionActionsListWithApi'
+import AnswerSummaryButton from './AnswerSummaryButton'
+import { useParticipant } from '../globals/contexts'
+
+type Props = {
+    questions: Question[]
+    viewProgression: Progression
+    disable: boolean
+    displayActions?: boolean
+    useFacilitatorAnswer?: boolean
+    onQuestionSummarySelected?: (question: Question, questionNumber: number) => void
+}
+
+const QuestionsList = ({
+    questions,
+    viewProgression,
+    disable,
+    onQuestionSummarySelected,
+    displayActions = false,
+    useFacilitatorAnswer = false,
+}: Props) => {
+    const { azureUniqueId: participantUniqueId } = useParticipant()
```

Comment:
It is misleading to say that it is the id of the participant. The participant id is the one in our database, not the one that is globally unique to that user. 

```suggestion
    const { azureUniqueId: currentUserAzureUniqueId } = useParticipant()
```

Reply:
I don't think this was my naming, but I can change it.